MAUI/Expander/Getting-Started.md

```diff
@@ -0,0 +1,400 @@
+	
+    <syncfusion:SfExpander.Content>
+        <Grid>
+            <Label Text="Veggie burger, garden burger, or tofu burger uses a meat analogue, a meat substitute such as tofu, textured vegetable protein, seitan (wheat gluten), Quorn, beans, grains or an assortment of vegetables, which are ground up and formed into patties."/>
+        </Grid>
+    </syncfusion:SfExpander.Content>
+</syncfusion:SfExpander> 
+{% endhighlight %}
+{% highlight c# %}
+expander.Collapsed += Expander_Collapsed;
+            
+private void Expander_Collapsed(object sender, ExpandedAndCollapsedEventArgs e)
+{
+    label.Text = "Burger and Pizza";
+    expander.Header.BackgroundColor = Color.Aqua;
+}
+{% endhighlight %}
+{% endtabs %}
+
+You can customize the expander by using the `Expanded` event. It will occur after a `SfExpander` is expanded when tapping on the header. It will provide the information related to the `Expanded` event by using the `ExpandedAndCollapsedEventArgs`.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfExpander x:Name ="expander" Expanded="Expander_Expanded">
+    <syncfusion:SfExpander.Header>
+        <Grid >
+            <Label x:Name="label"  Text="Veggie burger" FontSize="Large"/>
+        </Grid>
+    </syncfusion:SfExpander.Header>
+	
+    <syncfusion:SfExpander.Content>
+        <Grid>
+            <Label Text="Veggie burger, garden burger, or tofu burger uses a meat analogue, a meat substitute such as tofu, textured vegetable protein, seitan (wheat gluten), Quorn, beans, grains or an assortment of vegetables, which are ground up and formed into patties."/>
+        </Grid>
+    </syncfusion:SfExpander.Content>
+</syncfusion:SfExpander>
+{% endhighlight %}
+{% highlight c# %}
+expander.Expanded += Expander_Expanded;
+            
+private void Expander_Expanded(object sender, ExpandedAndCollapsedEventArgs e)
+{
+    label.Text = "Burger";
+    expander.Header.BackgroundColor = Color.YellowGreen;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Restricting the Expander while expanding and collapsing 
+
+You can prevent the Expander from being collapsed by handling the `Collapsing` event. This event occurs when a user attempts to collapse the `SfExpander` by tapping on the header. You can cancel the user action by setting the `Cancel` property of `ExpandingAndCollapsingEventArgs`.
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfExpander x:Name ="expander" Collapsing="Expander_Collapsing" >
+</syncfusion:SfExpander>
+{% endhighlight %}
+{% highlight c# %}
+expander.Collapsing += Expander_Collapsing;
+
+private void Expander_Collapsing(object sender, ExpandingAndCollapsingEventArgs e)
+{
+   e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+You can restrict the Expander being expanded by the `Expanding` event. This event occurs when an user tries to expand the `SfExpander` while tapping on the header. You can cancel the user action using the `Cancel` property of `ExpandingAndCollapsingEventArgs`.
```

Comment:
Restrict the Expander being expanded by the `Expanding` event. This event occurs when a user tries to expand the `SfExpander` while tapping the header. Cancel the user action using the `Cancel` property of the `ExpandingAndCollapsingEventArgs.`

Reply:
Addressed the concerns provided.